docs/usage/testdrive.mdx

```diff
@@ -4,19 +4,28 @@ title: Isolating File Operations using the TestDrive
 sidebar_label: TestDrive
 ---
 
-TestDrive is a PowerShell PSDrive for file activity limited to the scope of a single Describe or Context block.
+TestDrive is a PowerShell PSDrive for file activity limited to the scope of a block ([Describe](../commands/Describe) or [Context](../commands/Context)).
 
-A test may need to work with file operations and validate certain types of file activities. It is usually desirable not to perform file activity tests that will produce side effects outside of an individual test. Pester creates a PSDrive inside the user's temporary drive that is accessible via a named PSDrive ```TestDrive:```. Pester will remove this drive after the test completes. You may use this drive to isolate the file operations of your test to a temporary store.
+A test may need to work with file operations and validate certain types of file activities. It is usually desirable not to perform file activity tests that will produce side effects outside of an individual test. Pester creates a PSDrive inside the user's temporary drive that is accessible via a named PSDrive ```TestDrive:```. Pester will remove this drive after the tests completes. You may use this drive to isolate the file operations of your test to a temporary store.
 
 ## Scoping
 
-Basic scoping rules are implemented for the TestDrive. A clean TestDrive is created for every [Describe](../commands/Describe) and all the files created are available in the whole [Describe](../commands/Describe) scope. If the [Context](../commands/Context) keyword is also used the state of the TestDrive is recorded before moving into the Context block. Inside the [Context](../commands/Context) block the files from the [Describe](../commands/Describe) scope are available for reading and modification. You can move them around and create new ones as well.
+Basic scoping rules are implemented for the TestDrive:
 
-Once the [Context](../commands/Context) block is finished all the files created inside that block are deleted, leaving only the files created in the Describe block. When the Describe block is finished all contents of the TestDrive are discarded.
+1. A clean TestDrive is created per container (test-file or scriptblock) on entry to the first top-level block (Describe or Context) in that container.
```

Comment:
I think this is for each top-level block in container. Not whole container.

Reply:
Might have to rephrase. 
The path for the TestDrive-root is the same for the whole container. The content however is cleared between each top-level block, as part of "exiting block" cleanup on any level. I tried to cover that part in a later step.

Reply:
Ok. Let me try it first.

Reply:
Sorry you are right. This probably changed with the introduction of root block. Root is now acting as the top-level block and all describes in the file share the same drive. The content is cleared, unless you add it in the top-level BeforeAll. In that case it stays. Which is correct, it is still the same logical behavior like when returning from any other child describe. The items that were created above the current level are still present, unless you removed them yourself in the child.